tests/lib/plugin.js

```diff
@@ -10,6 +10,31 @@ var assert = require("chai").assert,
     path = require("path"),
     plugin = require("../..");
 
+/**
+ * Helper function which creates CLIEngine instance with enabled/disabled autofix feature.
+ * @param {boolean} [isAutofixEnabled=false] Whether to enable autofix feature.
+ * @returns {CLIEngine} CLIEngine instance to execute in tests.
+ */
+function initCLI(isAutofixEnabled) {
```

Comment:
I feel like this function should be moved to its own file.

Reply:
Since this isn't used anywhere else right now, I have no complaints about leaving it right here where you put it 👍